gqlalchemy/query_builders/memgraph_query_builder.py

```diff
@@ -79,8 +80,109 @@ def load_csv(self, path: str, header: bool, row: str) -> "DeclarativeBase":
 
         return self
 
+    def construct_subgraph_query(
+        self, node_label: Optional[str] = None, relationship_types: Optional[Union[str, Iterable[str]]] = None
+    ) -> str:
+        """Constructs a MATCH query defining a subgraph using zero or one node labels and relationship types.
+
+        Args:
+            node_label (Optional[str], optional): node label to be used in subgraph
+            relationship_types (Optional[Union[str, Iterable[str]]], optional): types of relationships used in subgraph.
+
+        Returns:
+            str: a MATCH query for a path with given node labels and relationship types
```

Comment:
str is not variable name, but return type, so maybe just:
```suggestion
            A string that represents a MATCH query for a path with given node labels and relationship types.
```

So far, we followed [Google docstring](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings) standard I think, so it is enough to describe the return argument and add a period at the end of each line.